File: AndroidCourse/Java/Lesson_1/app/src/main/java/com/example/lesson_1/MainActivity.java
package com.example.lesson_1;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button buttonToAct1Main = findViewById(R.id.buttonToAct1Main);
        buttonToAct1Main.setOnClickListener(view -> startActivity(new Intent(MainActivity.this, Activity1.class)));

        Button buttonToAct2Main = findViewById(R.id.buttonToAct2Main);
        buttonToAct2Main.setOnClickListener(view -> startActivity(new Intent(MainActivity.this, Activity2.class)));
    }
}

File: AndroidCourse/Kotlin/Lesson_1/app/src/main/java/com/example/lesson_1/MainActivity.kt
package com.example.lesson_1

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        buttonToAct1Main.setOnClickListener {
            startActivity(Intent(this, Activity1::class.java))
        }

        buttonToAct2Main.setOnClickListener {
            startActivity(Intent(this, Activity2::class.java))
        }
    }
}